docs/ensnode.io/src/styles/starlight.css

```
@@ -452,15 +452,46 @@ ul.top-level details details > summary .group-label span {
   transform: none !important;
 }
 
-/* Align "tablist" component to the left
- * and remove the default full-width underline
- */
+/* Unify the styling of Starlight tabs with our Omnigraph Static example tabs */
 starlight-tabs ul[role="tablist"] {
-  padding-left: 0;
   border-bottom: none;
```

Stylelint (17.12.0)

[error] Duplicate property "border-bottom" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)